### ISSUE TITLE
Improve webauthn errors

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6157]
Problems:

- We currently have two pieces of JS that control the display of webauthn errors - some in `index.js`, some in `webauthn.js`
- The errors are not announced to the screenreader (via `role=alert`)
- The errors are not associated with the webauthn label field (on the provisioning page) - we should use `aria-describedby` for this
- The user is able to put text into the label field on the provisioning page - it should be disabled

**Comment by nlhkabu** [comment]
@woodruffw as most of this JS is yours, it would be helpful if we could look into this together?  Thanks

**Comment by woodruffw** [comment]
Yep! Let's set aside some time for this before ending the WebAuthn beta.
